### PR TITLE
update changelogs and package.jsons for 1.3.0

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

File: sqlite/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version 1.3.0 - 2023-10-05
+
+### Changed
+
+- Updated minimum required version of `@cap-js/db-service`
```
Comment (johannes-vogel) [inline sqlite/CHANGELOG.md:11]:
```suggestion
### Fixed 

- `CURRENT_TIMESTAMP` in view definition preserves the timezone
```

File: postgres/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
-### Version 1.3.0 - tbd
+### Version 1.3.0 - 2023-10-05
```
Comment (patricebender) [inline postgres/CHANGELOG.md:7]:
```suggestion
### Version 1.3.0 - 2023-10-06
```

File: sqlite/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version 1.3.0 - 2023-10-05
```
Comment (patricebender) [inline sqlite/CHANGELOG.md:7]:
```suggestion
## Version 1.3.0 - 2023-10-06
```

**Review by patricebender — COMMENTED** [review]
also in the `db-service/CHANGELOG.md` the date has to be updated.

**Review by patricebender — APPROVED** [review]

**Pull request merged**